source/platform/include/Gwork/Input/Irrlicht.h

```diff
@@ -0,0 +1,183 @@
+/*
+ *  Gwork
+ *  Copyright (c) 2011 Facepunch Studios
+ *  Copyright (c) 2013-17 Nick Trout
+ *  See license in Gwork.h
+ */
+#pragma once
+#include <irrlicht.h>
+#include <Gwork/InputEventListener.h>
+
+namespace Gwk
+{
+    namespace Input
+    {
+        class Irrlicht
+        {
+        protected:
+            IInputEventListener* m_eventListener;
+            irr::s32 MouseX;
```

Comment:
`m_mouseX` & `m_mouseY` for consistency? Or a vector2 type?

Reply:
I did it for consistency as the function call we pass these variables into doesn't accept a vector2 instead takes individual integers.